lib/style/linagora_hover_style.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:flutter/material.dart';
+import 'package:linagora_design_flutter/colors/linagora_sys_colors.dart';
+
+class LinagoraHoverStyle {
```

Comment:
how to use it?

Reply:
Apply the values to an inkwell widget

Reply:
can you demo it. still not see it in example